src/ImageRunModal.jsx

```diff
@@ -731,7 +731,12 @@ export class ImageRunModal extends React.Component {
                           labelIcon={
                               <Popover aria-label={_("Image selection help")}
                                 enableFlip
-                                bodyContent={_("host[:port]/[user]/container[:tag].")}>
+                                bodyContent={
+                                    <Flex direction={{ default: 'column' }}>
+                                        <FlexItem>{_("hostname[:port]/[user]/container[:tag]")}</FlexItem>
+                                        <FlexItem>{_("For example: quay.io/cockpit")}</FlexItem>
```

Comment:
Hah, clever! Can you please take out the container name from the translatable part, and also use a container name which actually exists?

    cockpit.format(_("Example: $0"), "quay.io/libpod/busybox")

That way, translators don't have to precisely copy&paste the container name, and we can re-use the translation for other examples in the future.

Reply:
Good point, did you mean to suggest that we change the example container search as well?

Reply:
You mean the first line in the tooltip on line 736? That seems fine to me, it gives the general  pattern.

Reply:
Your recent force push still has `quay.io/cockpit`, which doesn't exist; also, our own containers are not a particularly good example.

Reply:
ok, easy to resolve. But I wanted to give a "searching" example. so what about quay.io/busybox

Reply:
Oh, I see what you mean -- yes, that sounds fine then. It feels a little strange, as it's not a substring of the actual container names, but that's just the magic first slash :)

Reply:
I've now added two examples, for searching and specifying the whole image, what do you think?